pkg/cloud/services/ec2/routetables.go

```diff
@@ -17,9 +17,10 @@ limitations under the License.
 package ec2
 
 import (
-	"sigs.k8s.io/cluster-api/util/conditions"
 	"strings"
 
+	"sigs.k8s.io/cluster-api/util/conditions"
+
```

Comment:
This should be grouped with the below imports, and the two separate groups of imports below should likely also be grouped together